### ISSUE TITLE
Rule 0052 does not detect internal procedures being referenced with property syntax

### ISSUE THREAD

**Issue by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#518]
Today I learned you can invoke global procedures with property-syntax, e.g.:

```al
codeunit 70257126 MyCodeunitWithProperty
{
    var
        MyIntegerVariable: Integer;

    internal procedure MyInteger(NewIntegerValue: Integer)
    begin
        MyIntegerVariable := NewIntegerValue;
    end;
}

codeunit 70257127 SampleCodeunit
{
    var
        MyCodeunitWithProperty: Codeunit MyCodeunitWithProperty;

    trigger OnRun()
    begin
        MyCodeunitWithProperty.MyInteger := 42;
    end;
}
```

For this sample, rule 0052 does not detect that the internal procedure is referenced.

**Comment by pri-kise** [comment]
I think this is feature is limited to simple types like, Text, Integer, BigInteger and the procedure must only have one paramter.

**Comment by Arthurvdv** [comment]
Moved to ALCops
